Commands/Commands/queue.js

```diff
@@ -358,7 +358,10 @@ commands.registerVote = {
                     msg.addReaction({
                       id: '296752137935912960',
                       name: 'f1'
-                    }).then(setTimeout(() => msg.removeReaction({id: '296752137935912960', name: 'f1'}), 2500))
+                    }).then(() => {
+                      var muppetsArr = msg.fetchReactions({id: '296752137935912960', name: 'f1'}, bot.User) // muppetsArr => People trying to be funny by reacting to f1
```

Comment:
Wrong usage of the [`fetchReations()` function](http://qeled.github.io/discordie/#/docs/IMessage?p=IMessage%23fetchReactions&_k=cgymn2)